.github/meta_issue_template/release-1.1.1-component-issues-min.md

```diff
@@ -0,0 +1,55 @@
+This is a component issue for release 1.1.1.
+Coming from [release issue 1.1.1](https://github.com/opensearch-project/opensearch-build/issues/870), release version 1.1.1. Please follow the following checklist.
+
+<details><summary>How to use this component issue</summary>
+<p>
+
+## This Component Issue
+This component issue captures the state of the OpenSearch release, on component/plugin level, its assignee is responsible for driving the release of the component.  Please contact them or @mention them on this issue for help. 
+
+## Release Steps
+There are several steps to the release process, components that are behind present risk to the release.  Component owners resolve tasks on this ticket to communicate with the overall release owner.
+
+Steps have completion dates for coordinating efforts between the components of a release; components can start as soon as they are ready far in advance of a future release.
+
+You can find all the corresponding dates of each step in the release issue above.
+
+</p>
+</details>
+
+
+### You can find all the date in above issue 
+
+### Preparation
+
+- [ ] Assign this issue to a release owner.
+- [ ] All the tasks in this issue have been reviewed by the release owner.
+- [ ] Create, update, triage and label all features and issues targeted for this release with v1.1.1.
+
+### What should I do if my plugin isn't making any changes?
```

Comment:
Note; I mean to pull this into the `How to use this issue section` not to create a new section in the checklist with instructions of how to use the checklist

Reply:
Added.